src/hotspot/share/runtime/arguments.cpp

```diff
@@ -333,6 +334,29 @@ bool Arguments::is_internal_module_property(const char* property) {
   return false;
 }
 
+ccstr Arguments::process_nmt_property(JavaVMInitArgs* args) {
```

Comment:
Please don't use "property" here as "property" has a very specific meaning when it comes to command-line arguments. I think "setting" would suffice here.

Reply:
Fixed.